test/integration/docker_exec_websocket_server_test.js

```diff
@@ -0,0 +1,92 @@
+var DockerWorker = require('../dockerworker');
+var TestWorker = require('../testworker');
+var assert = require('assert');
+var cmd = require('./helper/cmd');
+var debug = require('debug')('docker-worker:test:docker_exec_websocket_server_test')
+// var waitForEvent = require('../../lib/wait_for_event');
+var slugid = require('slugid');
+var DockerExecClient = require('docker-exec-websocket-server').DockerExecClient;
+var request = require('superagent-promise');
+var Promise = require('promise');
+
+suite('use docker exec websocket server', () => {
+  let worker;
+  setup(async () => {
+    worker = new TestWorker(DockerWorker);
+    await worker.launch();
+  });
+
+  teardown(async () => {
+    if (worker) {
+      await worker.terminate();
+      worker = null;
+    }
+  });
+  test('cat', async () => {
+    let taskId = slugid.v4();
+  	let task = {
+      payload: {
+        image: 'taskcluster/test-ubuntu',
```

Comment:
I think you'll like the `busybox` image more, it's really just 5mb :) and can sleep too...